pymedphys/labs/quickcheck/qcheck.py

```diff
@@ -78,11 +78,11 @@ def send_quickcheck(self, message):
                     )
                     self.data = ""
                     break
-                else:
-                    print("Connection Timeout")
-                    n_retry += 1
-                    print("Retrying connection {}/{}".format(n_retry, max_retries))
-                    continue
+
+                print("Connection Timeout")
+                n_retry += 1
+                print("Retrying connection {}/{}".format(n_retry, max_retries))
+                continue
```

Comment:
Is this continue needed here?

Reply:
mmm, actually is not. 
I finally managed to pass pylint checks. I installed it with pycharm.  The problem was it was throwing some warnings and a recommendation and I thought there was something else that was failing. I am pushing another commit without the continue statement.

Reply:
Thanks for your tips, by the way

Reply:
My pleasure :). Sorry there are so many steps involved in getting something pushed in. ... in the long run I find these nit-picky tools do make large code bases easier to maintain. So worth it in the end.